### PR TITLE
Updates metrics plugins and tests for Node 20

### PR DESCRIPTION
**Pull request by colincasey**
[W-13067131](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001Pp1ZpYAJ/view)

### CODE REVIEW

File: plugin/download.sh [hunk 1]
```diff
@@ -64,6 +66,10 @@ download_assets_for_release() {
     # Node 19
     download "https://github.com/heroku/heroku-nodejs-plugin/releases/download/$tag/heroku-nodejs-plugin-node-19-$tag.sha512" "$dir/heroku-nodejs-plugin-node-19.sha512"
     download "https://github.com/heroku/heroku-nodejs-plugin/releases/download/$tag/heroku-nodejs-plugin-node-19-$tag.tar.gz" "$dir/heroku-nodejs-plugin-node-19.tar.gz"
+
+    # Node 20
+    download "https://github.com/heroku/heroku-nodejs-plugin/releases/download/$tag/heroku-nodejs-plugin-node-19-$tag.sha512" "$dir/heroku-nodejs-plugin-node-20.sha512"
```
Comment (Malax) [inline plugin/download.sh:71]: Is it correct that we download the node 19 version for node 20? Are they compatible and don't warrant a new plugin?

**Review by Malax — APPROVED** [review]
I didn't take a close look at the test fixtures as they look like being copied from an older version, if the answer to my one comment is "yes", ship it! 🚢

File: plugin/download.sh (hunk 1 above)
Comment (colincasey) [reply plugin/download.sh:71]: no, i missed updating a 19 to 20 here.

Comment (colincasey) [reply plugin/download.sh]: fixed in latest commit

**Pull request merged**